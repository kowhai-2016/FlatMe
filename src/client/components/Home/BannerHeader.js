import React from 'react'

const BannerHeader = () => {
  return (
    <div className='header'>
      <div className='container description'>
        <h1>FlatMe</h1>
        <h3>resolves all your flatting issues</h3>
        <hr />
        <h2>Make the Rules, Share the load, Live the dream!</h2>
        <h3>Creates transparency, accountability and harmoney between flatmates to maintain ideal living conditions</h3>
      </div>
    </div>
  )
}

export default BannerHeader
