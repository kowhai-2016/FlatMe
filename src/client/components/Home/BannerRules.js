import React from 'react'

const BannerRules = () => {
  return (
    <div className='container'>
      <h1>"Make the Rules"</h1>
      <p>contract generator at its finest</p>
    </div>
  )
}

export default BannerRules
