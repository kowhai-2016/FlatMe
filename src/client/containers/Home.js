import { connect } from 'react-redux'

import Home from '../components/Home'

const mapStateToProps = state => {
  return {
    account: state.account
  }
}

export default connect(mapStateToProps)(Home)
