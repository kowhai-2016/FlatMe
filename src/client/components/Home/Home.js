import React from 'react'

import BannerHeader from './BannerHeader'
import BannerRules from './BannerRules'

const Home = () => {
  return (
    <div>
      <BannerHeader />
      <BannerRules />
    </div>
  )
}

export default Home
