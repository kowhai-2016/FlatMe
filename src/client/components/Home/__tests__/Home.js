/* eslint-env jest */

import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'
import React from 'react'

import Home from '..'

test('Home renders correctly', () => {
  const wrapper = shallow(<Home />)
  expect(toJSON(wrapper)).toMatchSnapshot()
})
