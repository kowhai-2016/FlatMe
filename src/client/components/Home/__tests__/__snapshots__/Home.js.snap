exports[`test Home renders correctly 1`] = `
<div>
  <BannerHeader />
  <BannerRules />
</div>
`;
